src/components/dataDisplays/RelationshipDisplay.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import DataDisplay from './DataDisplay';
+import ActionIcon from '../ActionIcon';
+
+export default function RelationshipDisplay({
+  data,
+  loading,
+  setConfirmDeleteDialog,
+  ...rest
+}) {
+  const relationshipCols = [
+    {
+      reference: 'nonSelfFirstName',
+      name: 'nonSelfFirstName',
+      label: 'Individual',
```

Comment:
You can change all of these `label`s to `labelId`s.

Reply:
Done